### PR TITLE
update cli menu in sidebar

### PR DESCRIPTION
**Pull request by thisisjofrank**
Remove env variables from CLI section (same content is covered in references)
Reorder "all commands" list to match --help output
Update CLI sidebar order
Move LSP integration file into references now that its sidebar item is under Reference not CLI

### CODE REVIEW

File: runtime/_data.ts [hunk 1]
```diff
@@ -46,11 +46,6 @@ export const sidebar = [
       {
         label: "Deno CLI",
         items: [
-          "/runtime/reference/cli/unstable_flags/",
-          {
-            label: "Environment Variables",
-            id: "/runtime/reference/cli/env_variables/",
```
Comment (marvinhagemeister) [inline runtime/_data.ts:52]: Will this page be completely removed from the sidebar?

File: runtime/reference/cli/all_commands.md [hunk 2]
```diff
@@ -8,7 +8,23 @@ runtime environment from your terminal or command prompt. The CLI has a number
...
+- [deno run](/runtime/reference/cli/run/) - run a script
+- [deno serve](/runtime/reference/cli/serve/) - run a web server
+- [deno task](/runtime/reference/cli/task_runner/) - run a task
+- [deno repl](/runtime/reference/cli/repl/) - starts a read-eval-print-loop
+- [deno eval](/runtime/reference/cli/eval/) - evaluate provided script
+
+## Dependency management
+
 - [deno add](/runtime/reference/cli/add) - Add dependencies
+- [deno install](/runtime/reference/cli/install/) - install a dependency or a
```
Comment (lucacasonato) [inline runtime/reference/cli/all_commands.md:22]: or a ...

File: runtime/reference/cli/all_commands.md [hunk 3]
```diff
@@ -20,27 +36,20 @@ below for more information on each subcommand.
...
 - [deno doc](/runtime/reference/cli/documentation_generator/) - generate
   documentation for a module
-- [deno eval](/runtime/reference/cli/eval/) - evaluate provided script
 - [deno fmt](/runtime/reference/cli/formatter/) - format your code
 - [deno info](/runtime/reference/cli/dependency_inspector/) - inspect an ES
   module and all of its dependencies
-- [deno init](/runtime/reference/cli/init/) - create a new project
-- [deno install](/runtime/reference/cli/install/) - install a dependency or a
-  script
+- [deno init](/runtime/reference/cli/init/) - create a new project script
```
Comment (lucacasonato) [inline runtime/reference/cli/all_commands.md:42]: I don't know the last thing is better than the first

File: runtime/reference/cli/all_commands.md [hunk 4]
```diff
@@ -20,27 +36,20 @@ below for more information on each subcommand.
...
 - [deno jupyter](/runtime/reference/cli/jupyter/) - run a Jupyter notebook
 - [deno lint](/runtime/reference/cli/linter/) - lint your code
 - [deno lsp](/runtime/reference/cli/lsp/) - language server protocol integration
 - [deno publish](/runtime/reference/cli/publish/) - publish a module
```
Comment (lucacasonato) [inline runtime/reference/cli/all_commands.md:46]:
```suggestion
- [deno publish](/runtime/reference/cli/publish/) - publish a module to JSR
```

File: runtime/reference/cli/all_commands.md [hunk 5]
```diff
@@ -20,27 +36,20 @@ below for more information on each subcommand.
...
-- [deno remove](/runtime/reference/cli/remove) - Remove dependencies
-- [deno repl](/runtime/reference/cli/repl/) - starts a read-eval-print-loop
-- [deno run](/runtime/reference/cli/run/) - run a script
-- [deno serve](/runtime/reference/cli/serve/) - run a web server
-- [deno task](/runtime/reference/cli/task_runner/) - run a task
 - [deno test](/runtime/reference/cli/test/) - run your tests
-- [deno types](/runtime/reference/cli/types/) - print runtime types
-- [deno uninstall](/runtime/reference/cli/uninstall/) - uninstall a dependency
-  or a script
+- [deno types](/runtime/reference/cli/types/) - print runtime types or a script
```
Comment (lucacasonato) [inline runtime/reference/cli/all_commands.md:48]: or a script?

File: runtime/_data.ts (hunk 1 above)
Comment (thisisjofrank) [reply runtime/_data.ts:52]: yes

Comment (thisisjofrank) [reply runtime/_data.ts:52]: it is covered in the same named page that is in references

File: runtime/reference/cli/all_commands.md (hunk 5 above)
Comment (thisisjofrank) [reply runtime/reference/cli/all_commands.md:48]: urgh, linebreaks got me

File: runtime/reference/cli/all_commands.md (hunk 3 above)
Comment (thisisjofrank) [reply runtime/reference/cli/all_commands.md:44]: not sure what this is in reference to?
Moving the unstable flags to the bottom of the list?

Comment (lucacasonato) [reply runtime/reference/cli/all_commands.md]: "create a new project" vs "create a new project script"

Comment (thisisjofrank) [reply runtime/reference/cli/all_commands.md]: oh the script shouldnt be there, it was orphaned from another line, should be fixed now

**Review by marvinhagemeister — APPROVED** [review]

**Pull request merged**